src/settings/ExpenseClassSettings/ExpenseClassSettings.test.js

```diff
@@ -17,12 +18,17 @@ jest.mock('@folio/stripes-smart-components/lib/ControlledVocab', () => {
   return MockComponent;
 });
 
-const renderExpenseClassSettings = () => render(<ExpenseClassSettings stripes={{}} />);
+const stripesMock = { hasPerm: jest.fn(() => true) };
```

Comment:
if you are not going to test `hasPerm` method then it should be just func - `() => true`, jest mock is not needed here